pkg/utils/selector.go

```diff
@@ -181,8 +182,9 @@ func CheckPodMeetSelector(pod v1.Pod, selector v1alpha1.SelectorSpec) (bool, err
 	if err != nil {
 		return false, err
 	}
+
 	pods = filterByAnnotations(pods, annotationsSelector)
-	pods = filterByPhase(pods, v1.PodRunning)
+
```

Comment:
how about adding a pod phase parameter? I don't think that removing the phase check is a good idea.